From: Benjamin Drung <bdrung@ubuntu.com>
Subject: [PATCH] fixes/perl

Install the pure-perl parts of the perl bindings to the right location.

Signed-off-by: Benjamin Drung <bdrung@ubuntu.com>

---
 src/clients/lib/perl/wscript |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/clients/lib/perl/wscript b/src/clients/lib/perl/wscript
index cc9a9ed..3b69dea 100644
--- a/src/clients/lib/perl/wscript
+++ b/src/clients/lib/perl/wscript
@@ -17,9 +17,9 @@ def build(bld):
     obj.mac_bundle = True
 
     bld.install_files('${INSTALLDIR_PERL_LIB}', 'pm/XMMSClient.pm')
-    bld.install_files('${INSTALLDIR_PERL_LIB}', 'pm/XMMSClient/Sync.pm')
-    bld.install_files('${INSTALLDIR_PERL_LIB}', 'pm/XMMSClient/Glib.pm')
-    bld.install_files('${INSTALLDIR_PERL_LIB}', 'pm/XMMSClient/Collection.pm')
+    bld.install_files('${INSTALLDIR_PERL_LIB}/XMMSClient', 'pm/XMMSClient/Sync.pm')
+    bld.install_files('${INSTALLDIR_PERL_LIB}/XMMSClient', 'pm/XMMSClient/Glib.pm')
+    bld.install_files('${INSTALLDIR_PERL_LIB}/XMMSClient', 'pm/XMMSClient/Collection.pm')
 
 def configure(conf):
     conf.check_tool('perl')
-- 
tg: (947d326..) fixes/perl (depends on: master)
